[2025-03-01T06:27:04.749+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-03-01T06:24:00+00:00 [queued]>
[2025-03-01T06:27:04.758+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-03-01T06:24:00+00:00 [queued]>
[2025-03-01T06:27:04.758+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-03-01T06:27:04.771+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): load_to_postgres_file> on 2025-03-01 06:24:00+00:00
[2025-03-01T06:27:04.777+0000] {standard_task_runner.py:60} INFO - Started process 2718 to run task
[2025-03-01T06:27:04.780+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'crypto_etl_dag', 'load_to_postgres_file', 'scheduled__2025-03-01T06:24:00+00:00', '--job-id', '331', '--raw', '--subdir', 'DAGS_FOLDER/crypto_etl_dag.py', '--cfg-path', '/tmp/tmph6085s6s']
[2025-03-01T06:27:04.783+0000] {standard_task_runner.py:88} INFO - Job 331: Subtask load_to_postgres_file
[2025-03-01T06:27:04.831+0000] {task_command.py:423} INFO - Running <TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-03-01T06:24:00+00:00 [running]> on host 335ace54b661
[2025-03-01T06:27:04.906+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_etl_dag' AIRFLOW_CTX_TASK_ID='load_to_postgres_file' AIRFLOW_CTX_EXECUTION_DATE='2025-03-01T06:24:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-01T06:24:00+00:00'
[2025-03-01T06:27:04.909+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-01T06:27:04.911+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/scripts/load_to_postgres.py']
[2025-03-01T06:27:04.922+0000] {subprocess.py:86} INFO - Output:
[2025-03-01T06:27:05.910+0000] {subprocess.py:93} INFO - [INFO] Đã tải file '2025/03/01/13-27-03.csv' từ MinIO
[2025-03-01T06:27:05.910+0000] {subprocess.py:93} INFO - [INFO] Đã nạp 10 bản ghi vào Data Warehouse thành công.
[2025-03-01T06:27:05.911+0000] {subprocess.py:93} INFO - [INFO] Quá trình ETL hoàn tất. Đã xử lý 1 file mới từ MinIO.
[2025-03-01T06:27:05.979+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-01T06:27:06.000+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=crypto_etl_dag, task_id=load_to_postgres_file, execution_date=20250301T062400, start_date=20250301T062704, end_date=20250301T062706
[2025-03-01T06:27:06.037+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-01T06:27:06.058+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
